updatecli/updatecli.d/docker-images/404.yaml

```diff
@@ -39,46 +39,18 @@ targets:
       file: "config/ext_private-nginx-ingress_prodpublick8s.yaml"
       key: "defaultBackend.image.tag"
     scmid: default
-  updatePublicNginxIngress404ProdPublicK8s:
-    name: "Update 404 docker image tag"
-    kind: yaml
-    spec:
-      file: "config/ext_public-nginx-ingress_prodpublick8s.yaml"
-      key: "defaultBackend.image.tag"
-    scmid: default
   updatePrivateNginxIngress404TempPrivateK8s:
     name: "Update 404 docker image tag"
     kind: yaml
     spec:
       file: "config/ext_private-nginx-ingress_temp-privatek8s.yaml"
       key: "defaultBackend.image.tag"
     scmid: default
```

Comment:
```suggestion
```

Reply:
(Unless I missed something?)

Reply:
I didn't regrouped (yet?) the private nginx ingress config, only the public one.